File: hw2/codes/gon_furkan.py
for i in range(5):
    print("------------------------------")
    # shows unlabeled exit
    # break statement exits only one loop
    for j in range(5):
        if j==1:
            break
        print("innerLoopBreakUnlabeled: ", j)
    
    print("------------------------------")


    # no labeled exit in python
    
    # shows unlabeled continue 
    # continue statement affects only one loop
    for j in range(5):
        if j==2:
            continue
        print("innerLoopContinueUnlabeled: ", j)
    
    print("------------------------------")

    # no labeled continue in python

    # no labeled exit in python

File: hw2/codes/gon_furkan.lua
for i = 0, 5, 1 
do
    print("-------------------------------")
    
    -- shows unlabeled exit
    -- break statement exits only one loop
    for j = 0, 4, 1 
    do
        if j==1 then break end
        print("innerLoopBreakUnlabeled: ", j)
    end
     
     
    -- no labeled exit in Lua
     
    print("-------------------------------")
     
    -- no continue in Lua
    -- no labeled exit in Lua    
end